config/common/test_suites/origin_fast.yml

```diff
@@ -3,5 +3,6 @@ children:
   - "test_pull_request_origin_check"
   - "test_pull_request_origin_integration"
   - "test_pull_request_origin_extended_conformance_install"
+  - "test_pull_request_origin_extended_conformance_install_upgrade"
```

Comment:
@smarterclayton just want to confirm that we _are_ committing to running six jobs per PR test now